soccer/gameplay/skills/dribble.py

```diff
@@ -88,13 +103,27 @@ def aimed(self):
         angle = self.robot.angle - (self.pos - self.robot.pos).angle()
         return angle < .1 and angle > -.1
 
+    def on_enter_setup(self):
+        self.move_vector = main.ball().pos - self._pos
+        self.move_point = self._pos + self.move_vector + (
+            self.move_vector.normalized() * .2)
```

Comment:
It could be simplified down to `self.move_point = main.ball().pos + self.move_vector.normalized() * .2` though